src/fsharp/symbols/Symbols.fs

```diff
@@ -781,9 +781,13 @@ and FSharpUnionCase(cenv, v: UnionCaseRef) =
         checkIsResolved()
         v.Range
 
+    member __.HasFields =
```

Comment:
This seems like it would be useful, but it's never used yet. I would make another PR with `HasFields` and a use-case for it.

Reply:
There are many FCS APIs that are used outside of this repo only. What kind of use case you want me to add?

Reply:
Gotcha. Perhaps a test would be fine for it.

Reply:
OK, I've extracted this change to another PR and I'll a test there.